Base/usr/share/man/man7/Shell-vars.md

```diff
@@ -40,10 +40,13 @@ Its default value is `~/.history`.
 `PROMPT` (environment)
 
 The value of this variable is used to generate a prompt, the following escape sequences can be used literally inside the value, and they would expand to their respective values:
-- `\\u` : the current username
-- `\\h` : the current hostname
-- `\\w` : a collapsed path (relative to home) to the current directory
-- `\\p` : the string '$' (or '#' if the user is 'root')
+- `\\a`: bell character (behavior depends on terminal)
+- `\\e`: escape character (`0x1b`)
+- `\\h`: the current hostname
+- `\\p`: the string '$' (or '#' if the user is 'root')
+- `\\u`: the current username
+- `\\w`: a collapsed path (relative to home) to the current directory
+- `\\X`: reset style (foreground and background color, etc)
```

Comment:
I've always thought that some of these feel like unnecessary duplication, why have `\\e` when the string can contain the escape `\e`? why add `\\X` when you can just `\e[0m`?
the only ones I see that are actually needed are h,p,u,w - the rest can just be written as normal escapes

Not a problem with this PR, just ranting about this thing